draft-ietf-quic-http.md

```diff
@@ -954,7 +954,7 @@ might have acted on.
 A client that is unable to retry requests loses all requests that are in flight
 when the server closes the connection.  An endpoint MAY send multiple GOAWAY
 frames indicating different identifiers, but MUST NOT increase the identifier
-value they send, since clients might already have retried unprocessed requests
+value they carry, since clients might already have retried unprocessed requests
 on another connection.
```

Comment:
```suggestion
MUST NOT be greater than the identifier in any previous frame, since clients
might already have retried unprocessed requests on another connection.
```